docs/resources/deployment-examples/example-fargate/scripts/copy_config.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+echo "ls /config"
+ls /config
+cp -v /config_files/stellar_wks.toml /anchor_config
+cp -v /config_files/anchor_config.yaml /anchor_config
+cp -v /config_files/reference_config.yaml /anchor_config
+cp -v /config_files/sep.sh /anchor_config
+cat /anchor_config/anchor_config.yaml
+cat /anchor_config/stellar_wks.toml
+cat /anchor_config/reference_config.yaml
+echo "ls /anchor_config"
+ls /anchor_config
+echo "hello world3"
+echo "tailing for eternity..."
+tail -f /dev/null
```

Comment:
Is purpose of this to just sleep forever? If so `sleep infinity` might be nicer as it won't generate any IO.

Reply:
actually intended to remove this completely as config volume is not required after copying config to shared volume. updating.